packages/IMAPClient-Core.package/ICEndPoint.class/instance/parseFetchHeaderSender.for..st

```diff
@@ -0,0 +1,9 @@
+protocol parse
+parseFetchHeaderSender: aHeaderChunk for: anEmail
+
+	| sender |
+	sender := ((aHeaderChunk collect: [ :each | each withBlanksTrimmed ]) joinSeparatedBy: ' ').
+	sender := (sender subStrings: ':') allButFirst joinSeparatedBy: ':'.
```

Comment:
why join them seperated by ':'? where in the from part is another : except after from?